tests/cloudpickle_test.py

```diff
@@ -2288,11 +2275,7 @@ def check_literal_equal(obj1, obj2):
         ]
         for obj in literal_objs:
             depickled_obj = pickle_depickle(obj, protocol=self.protocol)
-            if sys.version_info[:3] >= (3, 5, 3):
-                assert depickled_obj == obj
-            else:
-                # __eq__ does not work for Literal objects in early Python 3.5
-                check_literal_equal(obj, depickled_obj)
+            assert depickled_obj == obj
```

Comment:
It seems that the definition of `check_literal_equal` can also be deleted.

Reply:
Ah, good catch. I removed this and `check_final_equal` in `test_generic_extensions_final` which also isn't being used anymore